spec/views/hq/admin_users/new.html.erb_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper' 
+
+describe 'hq/admin_users/new.html.erb', type: :view do
+
+  before :each do
+    assign :admin_user, build_stubbed(:admin_user)
+  end
+
+  it 'renders the form, its fields and submit button' do
+    render
+    expect(view).to render_template(:new)
+    expect(rendered).to have_selector("form")
+    ["email", "password", "password_confirmation"].each do |field|
+      expect(rendered).to have_selector("input[id='admin_user_#{field}']")
+    end
+    expect(rendered).to have_selector("input[type='submit'][value='Create Admin User']")
+  end
+
+  it 'has a cancel button going to the index page' do
```

Comment:
That's the only other button there is. 
The submit button is tested in the previous test, where it checks that all the standard form elements (form tag, fields and submit button) are present.